src/libs/ReportUtils.ts

```diff
@@ -3375,64 +3350,57 @@ function shouldReportBeInOptionList(report, currentReportId, isInGSDMode, betas,
 
 /**
  * Attempts to find a report in onyx with the provided list of participants. Does not include threads, task, money request, room, and policy expense chat.
- * @param {Array<Number>} newParticipantList
- * @returns {Array|undefined}
  */
-function getChatByParticipants(newParticipantList) {
-    const sortedNewParticipantList = _.sortBy(newParticipantList);
-    return _.find(allReports, (report) => {
-        // If the report has been deleted, or there are no participants (like an empty #admins room) then skip it
-        if (
-            !report ||
-            _.isEmpty(report.participantAccountIDs) ||
-            isChatThread(report) ||
-            isTaskReport(report) ||
-            isMoneyRequestReport(report) ||
-            isChatRoom(report) ||
-            isPolicyExpenseChat(report)
-        ) {
-            return false;
-        }
+function getChatByParticipants(newParticipantList: number[]): OnyxEntry<Report> {
+    const sortedNewParticipantList = newParticipantList.sort();
```

Comment:
This will sort inplace. Need to do non-inplace sort

Reply:
this is how its done originally https://github.com/Expensify/App/blob/2c7062b917a2d31b709691166633f171dd1fce8c/src/libs/ReportUtils.js#L3380-L3383

Reply:
I don't think they sorted in place.